<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsWindow</class>
 <widget class="QWidget" name="SettingsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tablet Switch Settings</string>
  </property>
  <property name="windowIcon">
   <iconset theme="applications-system">
    <normaloff>.</normaloff>.</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="0">
    <widget class="QCheckBox" name="stylusOnlyOptionsCheckBox">
     <property name="text">
      <string>Enable Stylus only Option (Disables Touch Screen  Input, Trackpad, and Built in Keyboard)</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="tabletSwitchDeviceNameLabel">
     <property name="whatsThis">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;This device input string can usually be determined by examining the results of  &amp;quot;libinput list-devices&amp;quot; and then using &amp;quot;evtest&amp;quot; to examine devices until you find one that contains and event code &amp;quot;SW_TABLET_MODE&amp;quot;.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string>Tablet Switch Device Name</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="touchDeviceLabel">
     <property name="text">
      <string>Touch Screen Device Name</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QComboBox" name="tabletSwitchDeviceNameComboBox">
     <property name="whatsThis">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;This device input string can usually be determined by examining the results of &amp;quot;libinput list-devices&amp;quot; and then using &amp;quot;evtest&amp;quot; to examine devices until you find one that contains and event code &amp;quot;SW_TABLET_MODE&amp;quot;.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QComboBox" name="systemTrayIconThemeComboBox">
     <item>
      <property name="text">
       <string>Dark</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Light</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="systemTrayIconThemeLabel">
     <property name="text">
      <string>System Tray Icon Theme</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QComboBox" name="touchDeviceComboBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>systemTrayIconThemeComboBox</tabstop>
  <tabstop>tabletSwitchDeviceNameComboBox</tabstop>
  <tabstop>stylusOnlyOptionsCheckBox</tabstop>
  <tabstop>touchDeviceComboBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
